src/tests/conformance/post_handshake/cmd/transaction.rs

```diff
@@ -21,9 +21,12 @@ async fn c012_TXN_submit_txn_and_expect_to_receive_it() {
         .expect("unable to build the kmd instance");
     kmd.start().await;
 
+    let wallets = kmd.get_wallets().await.expect("couldn't get the wallets");
+    println!("a temporary log with wallets: {:?}", wallets);
```

Comment:
This is a debug or printing something by purpose?

Reply:
yeah, it's there for anyone who wants to test this functionality (I needed it).

like it states: it's a temporary log until we write the full test